suite-native/device-manager/src/components/DeviceManagerContent.tsx

```diff
@@ -78,19 +79,19 @@ export const DeviceManagerContent = () => {
             }
             onClose={() => setIsChangeDeviceRequested(false)}
         >
-            <VStack spacing="medium">
+            <VStack spacing={12}>
                 <DeviceList
                     isVisible={isChangeDeviceRequested || isPortfolioTrackerDevice}
                     onSelectDevice={handleSelectDevice}
                 />
                 {!isPortfolioTrackerDevice && (
-                    <>
+                    <View style={{ gap: 12, paddingTop: 24 }}>
```

Comment:
do not use Inline styles
```suggestion
                    <VStack spacing={12} paddingTop="large" />
```

Reply:
f72721d0be